order.js

```diff
@@ -280,3 +384,21 @@ Render.proof = (proof) => {
     </div>
   `;
 }
+
+function solveTimeString(date) {
+  var ampm = "am";
+  var digit = date.getHours();
+  if(digit > 12) {
+    ampm = "pm";
+  }
+  switch(digit) {
```

Comment:
better algo. switch isn't right for this.
`
var digit = `${date.getMinutes()}`; // converts to string
if (digit.length == 1) digit = '0'+ digit;
`
Another algo
`var digit = `${date.getMinutes() < 10 ? '0' : ''}${date.getMinutes()}`;`